doc/Overview/Reactive/How-tos/SimpleFeed.md

```diff
@@ -1,30 +1,31 @@
 ---
-uid: Overview.Reactive.HowTos.SimpleFeed
+uid: Overview.Reactive.SimpleFeed
```

Comment:
If we're adjusting the uid, can we replace Reactive with MVUX

Reply:
Furthermore, it was clashing with the already existing Reactive docs which we've moved to Reference.